AEM/mlperf/README.md

```diff
@@ -7,9 +7,720 @@ translation, and others. The benchmark is representative of real-world
 workloads and as a fair and useful way to compare the performance of different
 machine learning systems.
 
+In this document, we'll show how to run Intel MLPerf v3.1 submission with Intel
+optimized Docker images.
+
+## Intel Docker Images for MLPerf
+
+The Intel optimized Docker images for MLPerf v3.1 can be built using the Dockerfiles.
+Example for building docker image with Dockerfile:
+```
+cd <THIS_REPO>/closed/Intel/code/resnet50/pytorch-cpu/docker/
+
+bash build_resnet50_contanier.sh
+```
+
+## HW configuration:
+
+| System Info     | Configuration detail                 |
+| --------------- | ------------------------------------ |
+| CPU             | SPR                       |
+| OS              | CentOS  Stream 8                     |
+| Kernel          | 6.1.11-1.el8.elrepo.x86_64 |
+| Memory          | 1024GB (16x64GB 4800MT/s [4800MT/s]) |
+| Disk            | 1TB NVMe                             |
+
+Best Known Configurations:
+
+```
+sudo bash run_clean.sh
+```
+
+## Benchmarking using automation scripts
+
+For your convinience, we prepare a set of automation scritps to help you download data, create docker, do data and model preprocessing, run accuracy, performance and compliance test in a batch. Please refer to ./automation/README.md for details about the usage. 
+Example for using automation scripts:
+```
+cd <THIS_REPO>/closed/Intel/code/automation/
+
+# Download dataset
+bash download_dataset.sh resnet50 /data/mlperf_data
+
+# Test model performance
+PerformanceOnly="True" bash run.sh <model> <location> <postfix>
+```
+
+For more details, please refer to the instructions in <THIS_REPO>/closed/Intel/code/automation/README.md.
```

Comment:
you need to provide the repo link since this doc will be on IDZ instead of github

Reply:
I am adding the following contents, providing the repo link:

Note: All the codes and scripts are publicly accissible and can be downloaded from [GitHub](https://github.com/mlcommons/inference_results_v3.1). The following sessions will refer this GitHub repository as <THIS_REPO>.